Sources/Yams/Parser.swift

```diff
@@ -214,6 +261,11 @@ private extension Parser {
         try parse() // Drop YAML_STREAM_START_EVENT
     }
 
+    func startParse(with buffer: UnsafeBufferPointer<UInt8>) throws {
```

Comment:
```suggestion
    private func startParse(with buffer: UnsafeBufferPointer<UInt8>) throws {
```

Same with the other `startParse(...)` function.

Reply:
Those `private` modifier may be redundant since they are declared inside private extension of `Parser`. Swift 5.0 warns `public` modifier are redundant in public extension, but not on `private` modifier in private extension.

Reply:
Oh all good then!